### DISCUSSION TITLE
Possible enhancements (and help) to Gannt Mods

### DISCUSSION THREAD

**Discussion by f1li** [TIBCOSoftware/spotfire-mods#104] [Q&A]
Hi all and thanks for your hard work and support. I am trying to use the Gannt Mod but as it is now does not fit with my use case.

I have several lines (shorts alarms) with a certain begin-end pair. I can define 3 macro categories to group them but I don't want to consider the min start date and max end date because there might be holes in between (which means no alarms present). Is there a way to handle this? Otherwise I may keep them on distincted lines, but in this case is there any way to have the viz in chronological order?

See attached screen (Spotfire using Gannt Mods).
![immagine](https://user-images.githubusercontent.com/74111096/191299175-4d70ca22-c2ec-43b6-adb7-558d8a16630a.png)

This is what I need to get in Spotfire (made using Excel).
![immagine](https://user-images.githubusercontent.com/74111096/191304735-18ba5049-953e-46df-9e28-891c76a0ecf0.png)

Hope I was clear enough.
Many thanks in advance

filippo

**Comment by jleviaguirre** [comment]
what's the dataset you are using?
